runtime/compiler/control/rossa.cpp

```diff
@@ -1526,12 +1526,13 @@ onLoadInternal(
       if (TR::Compiler->target.cpu.getS390SupportsRI())
          ((TR_JitPrivateConfig*)(jitConfig->privateConfig))->hwProfiler = TR_ZHWProfiler::allocate(jitConfig);
 #elif defined(TR_HOST_POWER)
-#if !defined(J9OS_I5_V6R1) && !defined(J9OS_I5_V7R2) /* We may support it since i 7.3. */
+#if !defined(J9OS_I5_V6R1) && !defined(J9OS_I5_V7R2)  && !defined(J9OS_I5_V7R3) && !defined(J9OS_I5_V7R4) 
```

Comment:
Given we're at 4 OS versions now I strongly prefer to see this changed to a single `#if !defined(J9OS_I5)`.

Reply:
ok. After discussion with JVM developers, we would follow this rule, and disable all these kind of features by J9OS_I5. IBMi JVM team would follow following process in future:
1) For each new releases of IBM i , JVM developers need to check with IBMi HW/OS developers for these features availability. 
2) If no support still, nothing to do further. If they plan to enable, then JVM would add a new check here for supported releases(something like J9OS_I5_V7R5 or J9OS_I5_V7R6 ) .